app/viewmodels/checkAnswers/expenses/simplifiedExpenses/TotalExpensesSummary.scala

```diff
@@ -27,15 +27,15 @@ import viewmodels.implicits._
 
 object TotalExpensesSummary  {
 
-  def row(answers: UserAnswers)(implicit messages: Messages): Option[SummaryListRow] =
+  def row(answers: UserAnswers, taxYear: Int, businessId: String)(implicit messages: Messages): Option[SummaryListRow] =
     answers.get(TotalExpensesPage).map {
```

Comment:
Does this not need to get with Some(busienssId) or will that be added in the next subtask?